### DISCUSSION TITLE
el-descriptions 单元格怎么跨列啊？

### DISCUSSION THREAD

**Discussion by elarbor** [element-plus/element-plus#19246] [General]
![image](https://github.com/user-attachments/assets/93934669-1ad8-40b6-8bb1-c3e9ad6b4c34)

只有跨行？https://element-plus.org/zh-CN/component/descriptions.html#%E5%8D%95%E5%85%83%E6%A0%BC%E8%B7%A8%E8%A1%8C

**Comment by ntnyq** [comment]
跨列设置 `span` 属性不就可以了。
